continuous_integration/test_script.sh

```diff
@@ -29,8 +31,8 @@ else
 fi
 
 # Is directory still empty ?
-ls
+ls -ltra
```

Comment:
Maybe to check that the tests do not create temporary files in the local folder without deleting them at the end (e.g. joblib cache folder)?